lib/idx/authenticator/GeneralAuthenticator.ts

```diff
@@ -0,0 +1,24 @@
+import { Authenticator } from './Authenticator';
+
+export type GeneralAuthenticatorInputValues = {
+  verificationCode: string;
+};
+
+export class GeneralAuthenticator extends Authenticator {
```

Comment:
maybe rename this `PasscodeAuthenticator`? `GeneralAuthenticator` sort sounds like another base class

Reply:
How about `VerificationCodeAuthenticator`? to match with the input name. I think I'll also need to add some comment there.